pkg/graph/quarto.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 func WriteFileToBucket(ctx context.Context, quartoStoryID string,
-	file graphql.Upload) error {
+	file graphql.Upload,
```

Comment:
Maybe we should put the writing code to the client as well, but this can be done later

Reply:
I agree